### DISCUSSION TITLE
[Feature request] - Mono theme.  Make newsletter heading and description fields that can be user modified.

### DISCUSSION THREAD

**Discussion by bjazmoore** [GetPublii/Publii#1666] [Ideas]
Currently those two elements (newsletter and newsletterDesc) are defined in the mono.lang.json.  

![4](https://github.com/user-attachments/assets/da0727f1-af27-444c-9e1a-45440492762b)

Yes they can be changed there, but that makes it hard to use this theme for clients who will not know how to modify a json file should the theme be updated.

Please move these two fields to the config.json so that they are in the theme settings and modify the theme file partials/sidebar.hbs as needed.  Other files may need modified - I am not sure.

**Comment by bjazmoore** [comment]
I have edited the files that required changing to support the suggested revisions and created a version 0.3.1.0.

![5](https://github.com/user-attachments/assets/811978df-c094-435a-a06a-a7c0bd723d54)

Here is the new modified Mono theme:

[mono 0.3.1.0.zip](https://github.com/user-attachments/files/16881936/mono.0.3.1.0.zip)

**Reply by bjazmoore** [reply]
I think I got the version number wrong.

**Reply by candidexmedia** [reply]
Thank you for sharing your mod @bjazmoore !
